src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -259,6 +260,7 @@ def create(dataset, annotations=None, feature=None, model='darknet-yolo',
         # This large buffer size (8 batches) is an attempt to mitigate against
         # the SFrame shuffle operation that can occur after each epoch.
         'io_thread_buffer_size': 8,
+        'mlmodel_path': None,
```

Comment:
Huh, how come we don’t hit this exception: https://github.com/apple/turicreate/blob/386efa4eb5033573ee9120704a8c88a9a6151133/src/ml/neural_net/model_spec.cpp#L486

Either way, worth a TODO to have the Python side download the Core ML version of the pre-trained model from somewhere

Reply:
Because I pass in the mlmodel in create : `model = tc.object_detector.create(data, 'annotation', max_iterations=25, _advanced_parameters= {'mlmodel_path' : 'darknet.mlmodel', 'use_tensorflow' : True})` It was hitting that exception when I didn't pass the model path